src/Operators/finitedifference.jl

```diff
@@ -3622,8 +3622,13 @@ end
     idx,
     hidx,
 ) = field[]
-@inline getidx(parent_space, field::Fields.PointField, loc::Location, idx) =
-    field[]
+@inline getidx(
+    parent_space,
+    field::Fields.PointField,
+    loc::Location,
+    idx,
+    hidx,
+) = field[]
```

Comment:
Is this not a duplicate of lines 3618--3624?

Reply:
It is! I don't know how the one I changed could ever get called, though, I think all of these methods follow the same api, up to the `hidx`. I'll try removing it and see what CI says.

Reply:
This change only came along because I searched for any other mistakes following the first, and I noticed the different number of args.